beacon_node/eth2_libp2p/src/rpc/methods.rs

```diff
@@ -277,10 +277,7 @@ impl<T: EthSpec> RPCCodedResponse<T> {
 
     /// Tells the codec whether to decode as an RPCResponse or an error.
     pub fn is_response(response_code: u8) -> bool {
-        match response_code {
-            0 => true,
-            _ => false,
-        }
+        matches!(response_code, 0)
```

Comment:
This could just be `==`, but I guess it doesn't matter as long as Clippy is appeased

Reply:
Oh yeah. I just did what clippy told me, and moved on :joy_cat: